typesafe_conductr_cli/conduct_info.py

```diff
@@ -1,16 +1,20 @@
-from typesafe_conductr_cli import conduct_logging, conduct_url
+from typesafe_conductr_cli import conduct_url
+from typesafe_conductr_cli.conduct_logging import handle_connection_error, handle_http_error, pretty_json
 import json
 import requests
 
 
-# `conduct info` command
+@handle_connection_error
+@handle_http_error
 def info(args):
+    """`conduct info` command"""
+
     url = conduct_url.url('bundles', args)
     response = requests.get(url)
     response.raise_for_status()
 
     if (args.verbose):
-        conduct_logging.pretty_json(response.text)
+        pretty_json(response.text)
```

Comment:
Can you please revert to qualified name, i.e. `conduct_logging.pretty_son`? I think it's important to know where these functions come from.